starknet_devnet/server.py

```diff
@@ -111,5 +111,18 @@ def handle(error: StarkException):
     """Handles the error and responds in JSON. """
     return {"message": error.message, "status_code": error.status_code}, error.status_code
 
+@app.route('/api', methods = ['GET'])
+def api():
+    """Print available endpoints."""
+    routes = {}
+    # pylint: disable=protected-access
+    for url in app.url_map._rules:
+        routes[url.rule] = {}
+        routes[url.rule]["functionName"] = url.endpoint
+        routes[url.rule]["methods"] = list(url.methods)
+        routes[url.rule]["doc"] = app.view_functions[url.endpoint].__doc__
```

Comment:
I observed the output that we get for some endpoints. E.g. `/feeder_gateway/get_code` has this value under the `"doc"` property:
```
"\n    Returns the ABI and bytecode of the contract whose contractAddress is provided.\n    "
```
We could `.strip()` the `__doc__`